src/hotspot/share/classfile/systemDictionaryShared.hpp

```diff
@@ -188,6 +187,8 @@ class SystemDictionaryShared: public SystemDictionary {
                        outputStream* st) NOT_CDS_RETURN;
 
 public:
+  static bool check_for_exclusion_impl(InstanceKlass* k, bool silent);
```

Comment:
Since check_for_exclusion_impl is used only by CleanupDumpTimeLambdaProxyClassTable, I think it's better to keep it private, but declare CleanupDumpTimeLambdaProxyClassTable as a friend class of SystemDictionaryShared. We already do that for ExcludeDumpTimeSharedClasses.

Reply:
Fixed.